### PR TITLE
[SPARK-28806][DOCS][SQL] Document SHOW COLUMNS in SQL Reference

### PR DESCRIPTION
**Pull request by dilipbiswal**
### What changes were proposed in this pull request?
Document SHOW COLUMNS statement in SQL Reference Guide. 

### Why are the changes needed?
Currently Spark lacks documentation on the supported SQL constructs causing
confusion among users who sometimes have to look at the code to understand the
usage. This is aimed at addressing this issue.

### Does this PR introduce any user-facing change?
Yes. 

**Before:**
There was no documentation for this.

**After.**
Screen Shot 2019-09-02 at 11 07 48 PM
Screen Shot 2019-09-02 at 11 08 08 PM
Screen Shot 2019-09-02 at 11 11 45 PM

### How was this patch tested?
Tested using jykyll build --serve

### CODE REVIEW

**Comment by dilipbiswal** [comment]
cc @gatorsmile

File: docs/sql-ref-syntax-aux-show-columns.md [hunk 1]
```diff
@@ -18,5 +18,26 @@ license: |
...
+### Description
+Return the list of columns in a table. If the table does not exist, an exception is thrown.
 
-**This page is under construction**
+### Syntax
+{% highlight sql %}
+SHOW COLUMNS {IN | FROM} [db.]table {IN | FROM} database
+{% endhighlight %}
+**Note**
+- You can list the columns of a table in a database other than current database in one of following
```
Comment (albertusk95) [inline docs/sql-ref-syntax-aux-show-columns.md]: I think this should be `You can list the columns of a table in a database other than current database in one of the following ways:`

**Comment by gatorsmile** [comment]
LGTM

Thanks! Merged to master.

**Comment by dilipbiswal** [comment]
@gatorsmile Thank you very much for fixing the doc and merging in.

**Pull request closed without merging**